### ISSUE TITLE
delayed video

### ISSUE THREAD

**Issue by suganthikarthick** [EricssonResearch/openwebrtc#451]
Hi Robert,

It may be general question.
We are using the ARM based platform and we are using openwebrtc.
We are sending the audio and video from our target to a user viewing at browser.

Here, the video that we are sending does NOT have any slowess, which means that there is no issue in encoding in the targat platform.
Only issue is, at the starting, the video is going a 4 seconds delayed than audio. How to make the audio to go along with the video. How to make the sync in this case?

We are digging a lot for this issue.. Your suggestions will be valuable for us.
Thanks.

**Comment by superdump** [comment]
This is very likely the same issue as #447 and #448. We just need some time to investigate and fix it. Please bear with us.

**Comment by suganthikarthick** [comment]
Ok, Thanks.

**Comment by suganthikarthick** [comment]
Hi Robert,

When we tried to use the custom gst for the video, we found that there is no delay.
Currently we are using that.

In our target, when we are using 640,480, there is no delay. But with resolution 1280x720 and above, the delay is again coming and video is fully distorted. We are using IMX6 for our purpose.

With gst-launch command also, we are getting the similar performance.
Do you have any suggestion about how to increase the performance for higher resolution.

Thanks.

**Comment by godivyam** [comment]
any updates ?

**Comment by superdump** [comment]
Yes. A fix is working for H264 but when testing for VP8 with Firefox there is some odd delay behaviour. Still investigating.

**Comment by superdump** [comment]
Also, note that the receiving client is also responsible for A/V synchronisation.

**Comment by suganthikarthick** [comment]
Yes. A fix is working for H264 but when testing for VP8 with Firefox there is some odd delay behaviour. Still investigating.

Hi Robert,
What change we made for H264 to work better?
